src/workos.ts

```diff
@@ -33,7 +33,7 @@ import { SubtleCryptoProvider } from './common/crypto/subtle-crypto-provider';
 import { FetchHttpClient } from './common/net/fetch-client';
 import { IronSessionProvider } from './common/iron-session/iron-session-provider';
 
-const VERSION = '7.21.0';
+const VERSION = '7.21.1';
```

Comment:
Since this release will also include https://github.com/workos/workos-node/pull/1100, should this be a minor version bump instead? (`7.22.0`)

Reply:
Ah I didn't know that hadn't been released yet, will update this.